md/tests/test_normalization.py

```diff
@@ -71,6 +72,35 @@ def test_time_of_stop(self):
         for original, normalized in data:
             self.assertEqual(fix_TIME_OF_STOP(original), normalized)
 
+    def test_initial_year_cutoff(self):
+        # Check cutoff of initial year(s)
+        y0 = MD_FIRST_YEAR_TO_KEEP - 1
+        y1 = MD_FIRST_YEAR_TO_KEEP
+        y2 = MD_FIRST_YEAR_TO_KEEP + 1
+        orig_stops = pd.DataFrame({
+            'STOPDATE': [
+                '01/01/%d' % y0,
```

Comment:
If we're testing the edges, 01/01/y0 doesn't border any date in this test case. Wouldn't we want 01/01/y3, i.e. the day immediately following 12/31/y2?